### PR TITLE
Update code prepare for jquery new version

### PR DESCRIPTION
**Pull request by mrtuvn**
### Description (*)

Another collection files update for prepare for new version jquery
Continuing check other files 

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Questions or comments

CC: @sivaschenko can you review this

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Catalog/view/adminhtml/web/catalog/product-attributes.js [hunk 1]
```diff
@@ -26,15 +26,15 @@ define([
...
+            this.modal = $('#create_new_attribute').modal({
                 title: $.mage.__('New Attribute'),
                 type: 'slide',
                 buttons: [],
 
                 /** @inheritdoc */
                 opened: function () {
                     $(this).parent().addClass('modal-content-new-attribute');
-                    self.iframe = $('<iframe id="create_new_attribute_container">').attr({
+                    self.iframe = $('#create_new_attribute_container').attr({
```
Comment (sivaschenko) [inline app/code/Magento/Catalog/view/adminhtml/web/catalog/product-attributes.js]: It looks like the iframe element is not referenced but created here, so I don't think this change is correct

File: lib/web/fotorama/fotorama.js [hunk 2]
```diff
@@ -659,8 +659,8 @@ fotoramaVersion = '4.6.4';
 
     function readTransform(css, dir) {
         return css.match(/ma/) && css.match(/-?\d+(?!d)/g)[css.match(/3d/) ?
-                (dir === 'vertical' ? 13 : 12) : (dir === 'vertical' ? 5 : 4)
-                ]
+            (dir === 'vertical' ? 13 : 12) : (dir === 'vertical' ? 5 : 4)
```
Comment (sivaschenko) [inline lib/web/fotorama/fotorama.js]: Please do not edit built-in 3rd party libraries code

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the pull request @mrtuvn ! Please see my comments

**Comment by sivaschenko** [comment]
@mrtuvn please open the pull requests related to the Platform Health project (that includes JQuery upgrade) to `platform-health` branch and mark them with "Project: Platform Health" label. That will help to process them faster.

**Review by sivaschenko — APPROVED** [review]

**Comment by sivaschenko** [comment]
@magento run all tests

**Comment by sivaschenko** [comment]
Covered by existing functional tests

**Review by ihor-sviziev — APPROVED** [review]

**Review by Den4ik — APPROVED** [review]

**Pull request merged**